coco/state.py

```diff
@@ -3,12 +3,14 @@
 import hashlib
 import logging
 import os
+from pathlib import Path
```

Comment:
Just checking, we are running at least Python 3.6?

Reply:
Yeah, see https://github.com/chime-experiment/coco/blob/facb258e6dbf16852ab68cec55a593d86ddcdee1/scripts/coco#L1 and https://github.com/chime-experiment/coco/blob/facb258e6dbf16852ab68cec55a593d86ddcdee1/scripts/cocod#L1